docs/en/stack/ml/df-analytics/dfanalytics-limitations.asciidoc

```diff
@@ -67,8 +64,8 @@ results scales with increased training percentage.
 === Documents with missing values in analyzed fields are skipped
 
 If there are missing values in feature fields (fields that are subjects of the 
-{dfanalytics}), then the document that contains the fields with the missing 
-values will be skipped during the analysis.
+{dfanalytics}), then the document that contains these fields will be skipped 
```

Comment:
Not a big deal, but I generally try to use present tense instead of future tense:
```suggestion
{dfanalytics}), then the document that contains these fields is skipped 
```